en/documentation/index.md

```diff
@@ -239,8 +239,8 @@ These links were more prominent but haven't been updated in ages.
 [pickaxe]: https://pragprog.com/titles/ruby5/programming-ruby-3-3-5th-edition/
 [poodr]: https://www.poodr.com/
 [ruby-lsp]: https://github.com/Shopify/ruby-lsp
+[shopify_lsp]: https://shopify.github.io/ruby-lsp/editors.html
```

Comment:
The project isn't called `Shopify LSP` so let's change the name:

```suggestion
[ruby-lsp-supported-editors]: https://shopify.github.io/ruby-lsp/editors.html
```

Remember to update all the references to it too 🙂

Reply:
oops I missed it. fixed 070401deb